shell/platform/embedder/tests/embedder_test_context.h

```diff
@@ -72,9 +72,7 @@ class EmbedderTestContext {
 
   void SetRootSurfaceTransformation(SkMatrix matrix);
 
-  void SetRenderTargetType(
-      EmbedderTestBackingStoreProducer::RenderTargetType type,
-      FlutterSoftwarePixelFormat software_pixfmt);
```

Comment:
Unused method declaration with no definition. I removed it in https://github.com/flutter/engine/pull/56626 but missed cleaning this up.